docs/design-system/ui_kits/ovp/index.html

```
@@ -0,0 +1,600 @@
+              <ul class="list-tight tiny" style="padding-left:1rem;margin-top:6px">
+                <li>Stabilize <code>ovp-build-crystals</code> Stage B</li>
+                <li>Pack API v2 ergonomics</li>
+                <li>Reader vs. Maintainer split (BL-051)</li>
+              </ul>
+              <div class="muted tiny" style="margin-top:8px">Source: <code>00-Polaris/README.md</code></div>
+            </div>
+
+            <div class="card">
+              <h3 style="margin-top:0">Search</h3>
+              <form style="display:flex;gap:6px">
+                <input type="search" placeholder="Search the vault…" style="flex:1">
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)